### PR TITLE
Feat [admin] : changer status template

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets

- [#TicketID](https://dev.sun-asterisk.com/issues/xxx)

## What's this PR do ?

- [x] list review template

Screenshot from 2020-07-20 14-15-43

### CODE REVIEW

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr></table>

File: app/Http/Controllers/Admin/TemplateController.php [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+            ]);
+        }
+    }
+
+    public function changeStatus(Request $request, Template $template)
+    {
+        if ($request->status == TemplateStatus::STATUS_PUBLIC
+            || $request->status == TemplateStatus::STATUS_UNPUBLIC
+            || $request->status == TemplateStatus::STATUS_REVIEWING
+            ) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/TemplateController.php:152]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.IncorrectExact</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/TemplateController.php</br>**Line**: 154</br>Line indented incorrectly; expected 12 spaces, found 16

File: app/Http/Controllers/Admin/TemplateController.php [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                $result = $this->templateRepository->update($template->id, ['status' => $request->status]);
+                if ($result) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/TemplateController.php:154]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.IncorrectExact</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/TemplateController.php</br>**Line**: 156</br>Line indented incorrectly; expected 12 spaces, found 16

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr></table>

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr></table>

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr></table>

**Review by hatth-1632 — APPROVED** [review]

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr></table>

**Pull request merged**